sei/assetlist.json

```diff
@@ -44,11 +43,28 @@
             "symbol": "OIN",
             "logo_URIs": {
                 "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sei/images/oin.png"
-                
             },
             "coingecko_id": ""
+        },
+        {
+            "description": "ERIS liquid staked SEI",
+            "denom_units": [
+                {
+                    "denom": "factory/sei1x2fgaaqecvk8kwuqkjqcj27clw5p5g99uawdzy9sc4rku8avumcq3cky4k/ampSEI",
+                    "exponent": 0
+                },
+                {
+                    "denom": "ampSEI",
+                    "exponent": 6
+                }
+            ],
+            "base": "factory/sei1x2fgaaqecvk8kwuqkjqcj27clw5p5g99uawdzy9sc4rku8avumcq3cky4k/ampSEI",
+            "name": "ERIS Amplified SEI",
+            "display": "ampSEI",
+            "symbol": "ampSEI",
+            "logo_URIs": {
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sei/images/ampSEI.svg"
```

Comment:
for ampJUNO and ampSEI, the images are titled ampJuno and ampSei, but the image references have ampJUNO and ampSEI. Please note the discrepancy by the capital letters and realign the image names.